### PR TITLE
[Drogon] Add new port

### PR DESCRIPTION
**Pull request by an-tao**
This PR is adding the [Drogon](https://github.com/an-tao/drogon) framework.

### CODE REVIEW

File: ports/drogon/vcpkg.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index aab915d..4daeb87 100755
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.5)
+ project(drogon)
+ message(STATUS "compiler: " ${CMAKE_CXX_COMPILER_ID})
+ option(BUILD_CTL "Build drogon_ctl" ON)
+-option(BUILD_EXAMPLES "Build examples" ON)
```
Comment (BillyONeal) [inline ports/drogon/vcpkg.patch:9]: Shouldn't we pass this to `vcpkg_configure_cmake` rather than patching it?

Comment (an-tao) [reply ports/drogon/vcpkg.patch]: Thanks for your response, I've fixed this.

Comment (BillyONeal) [reply ports/drogon/vcpkg.patch]: Heh, thanks!

**Review by LilyWangL — APPROVED** [review]

**Comment by vicroms** [comment]
Thanks for the PR!

**Pull request merged**